### PR TITLE
HIVE-26623: Maintain same columns in COMPACTIONS view present in SYS …

### PR DESCRIPTION
**Pull request by SourabhBadhya**
…and INFORMATION_SCHEMA

### What changes were proposed in this pull request?

Maintain same columns in COMPACTIONS view present in SYS and INFORMATION_SCHEMA

### Why are the changes needed?

To maintain same columns across both the COMPACTIONS views

### Does this PR introduce _any_ user-facing change?

No

### How was this patch tested?

Existing unit tests modified.

### CODE REVIEW

File: metastore/scripts/upgrade/hive/upgrade-4.0.0-alpha-1-to-4.0.0-alpha-2.hive.sql [hunk 1]
```diff
@@ -276,4 +276,70 @@ JOIN
 WHERE
   SE.SCHEDULED_QUERY_ID=SQ.SCHEDULED_QUERY_ID;
 
+CREATE OR REPLACE VIEW `COMPACTIONS`
```
Comment (veghlaci05) [inline metastore/scripts/upgrade/hive/upgrade-4.0.0-alpha-1-to-4.0.0-alpha-2.hive.sql:279]: Please add HIVE ticket id as comment, see line 132.

Comment (SourabhBadhya) [reply metastore/scripts/upgrade/hive/upgrade-4.0.0-alpha-1-to-4.0.0-alpha-2.hive.sql:280]: Thanks for identifying this. Updated the PR.

**Review by veghlaci05 — APPROVED** [review]
LGTM, +1

File: metastore/scripts/upgrade/hive/upgrade-4.0.0-alpha-1-to-4.0.0-alpha-2.hive.sql [hunk 2]
```diff
@@ -276,4 +276,71 @@ JOIN
...
+  C_ERROR_MESSAGE,
+  C_NEXT_TXN_ID,
+  C_TXN_ID,
+  C_COMMIT_TIME,
+  C_HIGHEST_WRITE_ID,
+  C_INITIATOR_HOST,
+  C_INITIATOR_ID,
+  C_INITIATOR_VERSION,
+  C_CLEANER_START,
+  NVL(C_POOL_NAME, 'default'),
```
Comment (deniskuzZ) [inline metastore/scripts/upgrade/hive/upgrade-4.0.0-alpha-1-to-4.0.0-alpha-2.hive.sql:334]: same here

**Review by deniskuzZ — APPROVED** [review]
LGTM, minor optimization

File: metastore/scripts/upgrade/hive/hive-schema-4.0.0-alpha-2.hive.sql [hunk 3]
```diff
@@ -1947,12 +1952,17 @@ SELECT DISTINCT
...
+  C_NEXT_TXN_ID,
+  C_TXN_ID,
+  C_COMMIT_TIME,
   C_HIGHEST_WRITE_ID,
   C_INITIATOR_HOST,
   C_INITIATOR_ID,
   C_INITIATOR_VERSION,
   C_CLEANER_START,
-  NVL(C_POOL_NAME, 'default')
+  NVL(C_POOL_NAME, 'default'),
```
[the remark this answers is not in the document]
Comment (SourabhBadhya) [reply metastore/scripts/upgrade/hive/hive-schema-4.0.0-alpha-2.hive.sql]: Nice catch. Updated the PR.

File: metastore/scripts/upgrade/hive/upgrade-4.0.0-alpha-1-to-4.0.0-alpha-2.hive.sql (hunk 2 above)
Comment (SourabhBadhya) [reply metastore/scripts/upgrade/hive/upgrade-4.0.0-alpha-1-to-4.0.0-alpha-2.hive.sql]: Done.

**Pull request merged**